src/gui/screen_print_preview.cpp

```diff
@@ -88,6 +90,7 @@ GCodeInfo::GCodeInfo() {
     filament_type[0] = 0;
     filament_used_mm = 0;
     filament_used_g = 0;
+    valid_printer_settings = false;
```

Comment:
/question/ Does this mean, that if the gcode does not contain the comment, we will report to the user that it is not „fit“ their printer?

Reply:
That this gcode was set up for anoter printer. Yes.

Reply:
It should only pop up if there is this comment and it says its a different printer right?